src/main/lrsql/ops/query/generate.clj

```diff
@@ -0,0 +1,16 @@
+(ns lrsql.ops.query.generate
+  (:require [clojure.spec.alpha :as s]
+            [clojure.walk :as walk]
+            [lrsql.spec.reaction :as react-spec]))
+
+(s/fdef generate
+  :args (s/cat :cond-map (s/map-of :react-spec/condition-name
```

Comment:
when you use an alias keyword like this, you need double dots, like `::react-spec/condition-name`

Reply:
fixed